Automattic/woocommerce-payments
# Update padding on settings page cards.

Fixes #1950 

#### Changes proposed in this Pull Request

The padding applied to <CardBody> elements is added through a property named `size` which accepts things like "medium", "large", etc. When the value is "large", as is the case on the new settings page, it adds the class `is-size-large` and a consequent `padding: 24px 32px` CSS rule to the CardBody. 

https://github.com/Automattic/woocommerce-payments/blob/develop/client/settings/general-settings/index.js#L38

This PR adds some overriding CSS to set the padding to 24px all around based on design feedback.
It would have been nice to remove the usage of the `size` property completely but it has a default of `medium` if not set so we would have to override it anyways to get 24 px padding.

https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/card/body.js#L14

This should only apply to the Cards and CardBody elements on the settings page since it targets cards that are children of the `settings-section` class. 

#### Testing instructions

With this branch checked out and plugin recompiled, you should see a tighter alignment of the content within a settings section which lines up better with the first card on the page.

It's subtle but it's there 😄 

![Screen Shot 2021-06-02 at 4 53 44 PM](https://user-images.githubusercontent.com/6723003/120656331-d51adf00-c483-11eb-9673-92cb0039e4b9.png)

![Screen Shot 2021-06-02 at 4 53 36 PM](https://user-images.githubusercontent.com/6723003/120656335-d5b37580-c483-11eb-8d0d-46c541d181ed.png)

I don't see any tests covering this and it doesn't seem to me that a test needs to cover this change. All existing tests pass.
Not adding a changelog since it's just a minor display change.
Looks good on mobile.

-------------------
